Makefile

```diff
@@ -116,8 +117,13 @@ endif
 	go generate ./pkg/... ./cmd/...
 	./scripts/sync-up-helm-chart.sh
 
-push:
-	$(DOCKER) push $(REGISTRY)/$(TARGET):$(GIT_VERSION)
```

Comment:
This line should not be removed so that users can still push to whatever registry they're using when invoking `make push`. But it should **not** be run if running in the `after_success` `travis` phase.

We basically need to support `make push` for users, and for travis to build official tagged images. The idea is that we could use the same `push` target for both and just add checks within it for both cases. Feel free to split it out if it would help simplify things.

Reply:
Thank you @font !
Now, 
1. for users, not the Travis CI call the `make push`, it will keep the same with the original.
2. for Travis CI, it will do not push anything for PRs and will build and push official tagged images.